src/components/forms/DataTypes.ts

```diff
@@ -0,0 +1,48 @@
+import {SelectOptionWithKeyNameAndAbbreviation} from './SelectOptionInterface';
+
+
+export class DataTypes {
+    static CITE: SelectOptionWithKeyNameAndAbbreviation = {
+        key: 'CITE',
+        name: 'CITE-seq'
+    };
+    static HYB: SelectOptionWithKeyNameAndAbbreviation = {
```

Comment:
Doesn't typescript already enforce this type due to this field's inclusion in the array below? I thought that if the array is typed, typescript would enforce that its elements conform to the array type (at least for static objects like these).

Reply:
If the only access were through the list, that would be okay however, I don't expect the only access to be through the list due to conditional logic we have in validation rules that use these.